packages/dashboard/src/project/hook/hookEdit.tsx

```diff
@@ -20,22 +33,16 @@ import { HookFormAction } from './hookFormReducer';
 import { SlackHook } from './slackHook';
 import { TeamsHook } from './teamsHook';
 
-const TogglerWrapper = styled(HFlow)`
-  display: flex;
-  align-items: center;
-  cursor: pointer;
-  height: 3rem;
-`;
 const Toggler = ({ toggleExpanded, isExpanded, title }: any) => {
   return (
-    <TogglerWrapper onClick={() => toggleExpanded()} alignItems="center">
-      {isExpanded ? (
-        <Icon icon="angleDown" size={1} />
-      ) : (
-        <Icon icon="angleRight" size={1} />
-      )}
-      <Text fontWeight="bold">{title}</Text>
-    </TogglerWrapper>
+    <Box
+      style={{ display: 'flex', cursor: 'pointer' }}
+      onClick={() => toggleExpanded()}
+      alignItems="center"
+    >
```

Comment:
```suggestion
    <Box
      display="'flex"
      cursor="pointer"
      onClick={() => toggleExpanded()}
      alignItems="center"
    >
```

Reply:
Done, the cursor prop does not exist.
I styled the cursor with the sx prop.